src/Microsoft.OData.Core/MediaTypeUtils.cs

```diff
@@ -58,7 +58,7 @@ internal static class MediaTypeUtils
         /// <summary>
         /// Max size to cache match info.
         /// </summary>
-        private const int MatchInfoCacheMaxSize = 1024;
+        private const int MatchInfoCacheMaxSize = 256;

```

Comment:
Maybe keeping at 1024 is fine or making this customizable, so that customers can experiment without waiting for a new release.

Reply:
I 'd like to generate a nightly build for yogesh to verify the memory usage.

---
In reply to: [283560641](https://github.com/OData/odata.net/pull/1448#discussion_r283560641) [](ancestors = 283560641)